terraform/examples/daos_cluster/module.json

```diff
@@ -166,42 +204,66 @@
     {
       "name": "server_os_disk_type",
       "type": "string",
-      "description": "OS disk type e.g. pd-ssd, pd-standard",
+      "description": "OS disk type ie. pd-ssd, pd-standard",
       "default": "pd-ssd",
       "required": false
     },
     {
       "name": "server_os_family",
       "type": "string",
       "description": "OS GCP image family",
-      "default": null,
+      "default": "daos-server-centos-7",
       "required": false
     },
     {
       "name": "server_os_project",
       "type": "string",
-      "description": "OS GCP image project name",
+      "description": "OS GCP image project name. Defaults to project_id if null.",
       "default": null,
       "required": false
     },
+    {
+      "name": "server_pools",
+      "type": "list(object({\n    pool_name  = string\n    pool_size  = string\n    containers = list(string)\n    })\n  )",
+      "description": "If provided, this module will generate a script to create a list of pools. pool attributes have to be specified in a format acceptable by [dmg](https://docs.daos.io/v2.0/admin/pool_operations/) and daos.",
+      "default": [],
+      "required": false
+    },
     {
       "name": "server_preemptible",
       "type": "string",
-      "description": "If preemptible server instances",
-      "default": true,
+      "description": "If preemptible instances",
+      "default": false,
+      "required": false
+    },
+    {
+      "name": "server_service_account",
```

Comment:
What is gained by adding these Service Accounts?

Reply:
The previous way of specifying scopes was overly simple and would not work if a user tried to deploy DAOS without the default compute service account. That sa is often disabled by default so it is useful to allow specifying an sa. The scopes are part of that sa and can be used to restrict what actions can be taken. 
By specifying the sa e-mail to null, we are changing scopes to the default service account (same behavior as before).